src/date-range-input/date-range-input.tsx

```diff
@@ -405,6 +448,13 @@ export const DateRangeInput = ({
 
     const handleCalendarHover = (val: string) => {
         setHoverValue(val);
+        if (isFixedRangeSelection) {
+            if (val) {
+                actions.focus("start");
+            } else {
+                actions.focus("none");
+            }
+        }
```

Comment:
This seems to remove the focus indicator when there is no active hover. Shouldn't the start field always remain focused as long as the date picker is active?

Reply:
according to the figma, when there is no active hover the focus indicator should be removed

Reply:
as discussed, the indicator is meant to be visible on an active field